package.json

```diff
@@ -21,6 +21,8 @@
     "@commitlint/cli": "^12.0.0",
     "@commitlint/config-conventional": "12.1.1",
     "@octokit/rest": "18.5.2",
+    "@typescript-eslint/eslint-plugin": "^4.22.0",
+    "@typescript-eslint/parser": "^4.22.0",
```

Comment:
Just out of curiosity, where is this used? 🤔

Reply:
When we lint the CRA template. I prefer to have dev dependencies hoisted, so that - if we introduce some more ts - it's all on the same version